### DISCUSSION TITLE
Adguard home - password reset

### DISCUSSION THREAD

**Discussion by bloodyskullz** [AdguardTeam/AdGuardHome#3925] [Q&A]
Tried looking for this but I didn't find anything official from Adguard. Is there a way to reset the password for adguard home? I forgot mine.

Thanks

**Comment by ainar-g** [comment]
We have a Wiki page about it: <https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#password-reset>.

**Comment by bloodyskullz** [comment]
Reading the guide, I would be generating a new hash for my existing user and not a new one correct? Does this mean that the current yaml file already has a hash?

**Reply by bloodyskullz** [reply]
Never mind I see it. It does, I know what needs to be done now. I will report back and give it a try.

Thanks

**Comment by bloodyskullz** [comment]
Alright all is working now. Thank you for your assistance.

**Comment by swedenstyle34** [comment]
I can't set a password nor anything else, On my Asustor Drivestor AS1102T 2.

**Comment by 2nistechworld** [comment]
Hello

If you are using docker I found an easier way to reset the password, by using an apache docker image instead of installing apache2 on a server:

I used this command to call htpasswd from an apache  docker impage:

docker run httpd:2.4 htpasswd -B -n -b USERNAME PASSWORD

And then copy paste the password hash into your AdGuardHome.yaml and restart the adguard container.

**Comment by CamelT0E** [comment]
If you already have a Docker container in which the Apache Server is already running/Inside, you can just use it. I have several containers running and use Baikal:latest. This works great with every other container (with Apache2) but also!
